src/group_theory/abelianization.lean

```diff
@@ -107,3 +110,12 @@ theorem hom_ext (φ ψ : abelianization G →* A)
 monoid_hom.ext $ λ x, quotient_group.induction_on x $ monoid_hom.congr_fun h
 
 end abelianization
+
+/-- Equivalent groups have equivalent abelianizations -/
+def mul_equiv.abelianization_congr {G H : Type*} [group G] [group H] (e : G ≃* H) :
+  abelianization G ≃* abelianization H :=
+{ to_fun := abelianization.lift $ abelianization.of.comp e.to_monoid_hom,
+  inv_fun := abelianization.lift $ abelianization.of.comp e.symm.to_monoid_hom,
```

Comment:
Perhaps we should introduce `abelianization.map`?

Reply:
For the composition of `abelianization.lift` and `abelianization.of.comp`?